lib/recog/db_manager.rb

```diff
@@ -13,8 +13,12 @@ def initialize(path = DefaultDatabasePath)
   end
 
   def load_databases
-    Dir[self.path + "/*.xml"].each do |dbxml|
-      self.databases << DB.new(dbxml)
+    if File.directory?(self.path)
+      Dir[self.path + "/*.xml"].each do |dbxml|
```

Comment:
I've tweaked the current preference values a touch to spread out files where there might be overlap.